addons/mail/models/mail_message.py

```diff
@@ -1054,6 +1056,18 @@ def _to_store(self, store: Store, fields, *, format_reply=True, msg_vals=False,
                 "scheduledDatetime": scheduled_dt_by_msg_id.get(message.id, False),
                 "thread": Store.One(record, [], as_thread=True),
             }
+
+            def parse_incoming_emails(incoming_emails):
+                email_lst = []
+                for email in incoming_emails.split(","):
```

Comment:
You can use ``tools.mail.email_split_tuples`` which should find all emails in a given string (no need to split, parse, ...). It returns a list of tuples (name, email).